examples/dataflow-cloud-run/dataflowApp/src/main/java/com/demo/dataflow/GoBikeMainApp.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.demo.dataflow;
+
+import com.demo.dataflow.eventfn.ParseGoBikeEvents;
+import com.demo.dataflow.model.FailedMessage;
+import com.demo.dataflow.model.GoBike;
+import com.demo.dataflow.util.TableSchemaUtils;
+import com.google.api.services.bigquery.model.TableReference;
+import com.google.api.services.bigquery.model.TableRow;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.io.TextIO;
+import org.apache.beam.sdk.io.gcp.bigquery.BigQueryIO;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.PCollectionTuple;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class GoBikeMainApp {
```

Comment:
nit: This is potentially confusing with `MainApplication`(the cloud run spring app)
I'd just name this class `GoBikeToBigQuery` to fall more in line with the naming convention with the [beam examples](https://github.com/apache/beam/tree/master/examples/java/src/main/java/org/apache/beam/examples/complete) and the [cloud teleport dataflow templates](https://github.com/GoogleCloudPlatform/DataflowTemplates/tree/master/src/main/java/com/google/cloud/teleport/templates) Where the class name is very clearly just a functional description of what the pipeline does.